build/gulpfile.hygiene.js

```diff
@@ -361,6 +361,23 @@ function hygiene(some) {
 		});
 	});
 
+	const filelength = es.through(function (file) {
+
+		const fileName = path.basename(file.path);
+		const fileDir = path.dirname(file.path);
+		//check the filename is < 50 characters (basename gets the filename with extension).
+		if (fileName.length > 50) {
+			console.error(`File name in the path ${fileName} under ${fileDir} is too long. Rename file to have less than 50 characters.`);
+			errorCount++;
+		}
+		if (file.path.length > 255) {
```

Comment:
I'm not sure about this one - people may have their enlistments at different paths. What we'd really want to do is check the relative path from the root of the enlistment I would think if we're going to check this.